### DISCUSSION TITLE
Remove Photo app in favor of ImageGlass

### DISCUSSION THREAD

**Discussion by Tarek-Hasan** [Atlas-OS/Atlas#1247] [Suggestions]
Microsoft Photo app only supports few popular image formats and need additional 4 or 5 extensions to support few more formats like WebP, HEIF, AVIF etc. All of those image formats and 100+ more including RAW formats are supported out of the box by ImageGlass. It would be nice to have an option to install ImageGlass in favour of removing Photo app and its extensions.

**Comment by AudiEtoffe** [comment]
You can remove MS Photos and install Image Glass, that's what I did.  
Atlas won't package it as it adds bloat, plus its a 3rd party software they would need to navigate rights to, not to mention that 90% of users don't care.

**Comment by LumiFae** [comment]
Honestly, Atlas isn't the kind of project which would do something like this. This is a gaming OS and not an OS for everything, I think @AudiEtoffe makes a really good point that if you want something that atlas doesn't come with, just install it yourself. MS Photos can be easily uninstalled, if it wasn't able to be easily uninstalled it may be considered.
